hw03_finagnon.py

```diff
@@ -0,0 +1,288 @@
+#################################################################################
+# Author:Finagnon Adjagbodjou
+# Username: Fin
+#
+# Assignment: hw03
+# Purpose: Learn the purpose of pulling, pushing, etc.
+# Google Doc Link: https://docs.google.com/document/d/1BjZMQNg6d2PG-PlwE5gFo9XBrmMei0aY6B_CtXdtFLg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: 
+#
+#
+#################################################################################
+
+import turtle
+import random
+
+
+def rightturn(shape):
+    """
+    turns the turtle to the right
+    """
+    shape.right(90)
+    shape.forward(5)
+    shape.right(90)
+
+
+def leftturn(shape):
+    """
+    turns the turtle to the left
+    """
+    shape.left(90)
+    shape.forward(5)
+    shape.left(90)
+
+
+def createrectangle(shape):
+    """
+    creates the starting rectangle in which the turtle will move
+    """
+
+    shape.penup()
+    shape.pensize(20)
+    shape.setpos(-200, 250)
+    shape.pendown()
+    shape.color('black')
+    shape.forward(500)
+    shape.right(90)
+    shape.forward(505)
+    shape.right(90)
+    shape.forward(500)
+    shape.right(90)
+    shape.forward(500)
+
+
+def stamper(shape):
+    """
+    stamps the turtle's shape on the screen
+    """
+    shape.stamp()
```

Comment:
I made this comment before... this function does a single operation (`shape.stamp()`). So you don't need to write your own function to do it. Simply call the `.stamp()` method where you called `stamper` instead (lines 258, 260, and 262).